### PR TITLE
Python: Deploy Windows wheels

### PR DESCRIPTION
**Pull request by mdboom**

### CODE REVIEW

**Comment by mdboom** [comment]
@Dexterp37 : Would you mind testing this as a Windows user?  If you install the standard Windows Python distribution (3.5, 3.6 or 3.7), make sure it's on the PATH and run `pip3 install glean_sdk`, can you then start Python and `import glean`?

**Comment by Dexterp37** [comment]
> 
> 
> @Dexterp37 : Would you mind testing this as a Windows user? If you install the standard Windows Python distribution (3.5, 3.6 or 3.7), make sure it's on the PATH and run `pip3 install glean_sdk`, can you then start Python and `import glean`?

This doesn't seem to work:

> (base) C:\Users\Alessio>pip --version
> pip 19.3.1 from C:\Users\Alessio\Miniconda3\lib\site-packages\pip (python 3.7)
> 
> (base) C:\Users\Alessio>pip install glean_sdk
> Collecting glean_sdk
>   Downloading https://files.pythonhosted.org/packages/40/0e/c26b70d23ca835e5e944bdda9f6a6ba9309a9a7f83df4ee1a16f8f9a1ad7/glean_sdk-0.0.0-py3-none-any.whl (10.3MB)
>      |████████████████████████████████| 10.3MB 1.6MB/s
> Collecting cffi==1.13.1
>   Downloading https://files.pythonhosted.org/packages/e0/81/6d6433ebeebe92a5eef8159684a1f3959a180f74a008d7865c9f577e7bff/cffi-1.13.1-cp37-cp37m-win_amd64.whl (175kB)
>      |████████████████████████████████| 184kB ...
> Requirement already satisfied: pycparser in c:\users\alessio\miniconda3\lib\site-packages (from cffi==1.13.1->glean_sdk) (2.19)
> Installing collected packages: cffi, glean-sdk
>   Found existing installation: cffi 1.13.0
>     Uninstalling cffi-1.13.0:
>       Successfully uninstalled cffi-1.13.0
> Successfully installed cffi-1.13.1 glean-sdk-0.0.0
> 
> (base) C:\Users\Alessio>python3
> "python3" non è riconosciuto come comando interno o esterno,
>  un programma eseguibile o un file batch.
> 
> (base) C:\Users\Alessio>python
> Python 3.7.4 (default, Aug  9 2019, 18:34:13) [MSC v.1915 64 bit (AMD64)] :: Anaconda, Inc. on win32
> Type "help", "copyright", "credits" or "license" for more information.
> >>> import glean
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "C:\Users\Alessio\Miniconda3\lib\site-packages\glean\__init__.py", line 18, in <module>
>     from .glean import Glean
>   File "C:\Users\Alessio\Miniconda3\lib\site-packages\glean\glean.py", line 6, in <module>
>     from . import _ffi
>   File "C:\Users\Alessio\Miniconda3\lib\site-packages\glean\_ffi.py", line 22, in <module>
>     ffi.cdef(_load_header("glean.h"))
>   File "C:\Users\Alessio\Miniconda3\lib\site-packages\glean\_ffi.py", line 15, in _load_header
>     data = pkg_resources.resource_string(__name__, path).decode("utf-8")
>   File "C:\Users\Alessio\Miniconda3\lib\site-packages\pkg_resources\__init__.py", line 1157, in resource_string
>     self, resource_name
>   File "C:\Users\Alessio\Miniconda3\lib\site-packages\pkg_resources\__init__.py", line 1401, in get_resource_string
>     return self._get(self._fn(self.module_path, resource_name))
>   File "C:\Users\Alessio\Miniconda3\lib\site-packages\pkg_resources\__init__.py", line 1616, in _get
>     with open(path, 'rb') as stream:
> FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Alessio\\Miniconda3\\lib\\site-packages\\glean\\glean.h'
> >>>

**Comment by mdboom** [comment]
It looks like it's downloading glean 0.0.0 (which is an old test version from way back, expected to be broken).  So that means `pip` isn't matching on [the wheel I just cut yesterday for some reason](https://files.pythonhosted.org/packages/b2/37/2acaf2a3989aa9834d0444fbf8d6b21057a7e24092ddc9ca6d06ec1b14d0/glean_sdk-23.0.1-cp35-abi3-win_amd64.whl).  Let me look into this further...

**Comment by mdboom** [comment]
@Dexterp37 : I think I have it worked out.  Would you mind trying again by:

1) Uninstalling glean_sdk 0.0.0 with `pip uninstall glean_sdk`
2) Installing glean_sdk from the test PyPI instance (since PyPI rightfully won't let me upload a replacement package): `pip install -i https://test.pypi.org/simple glean_sdk`
3) Try to `import glean` again...

**Comment by Dexterp37** [comment]
After talking this over with Mike, we got it working. This PR is working just fine :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -237,14 +237,6 @@ jobs:
...
       xcode: "11.3.0"
     steps:
-      - run:
-          name: Set Ruby Version
-          command: echo 'chruby ruby-2.6.5' >> ~/.bash_profile
-      - run:
-          name: Show Ruby environment
-          command: |
-            ruby --version
-            gem env
```
Comment (mdboom) [inline .circleci/config.yml]: This is unrelated, but made it in unintentionally when adding macOS wheels.

**Pull request merged**